cmd/testscript/main.go

```diff
@@ -252,6 +224,13 @@ func run(runDir, filename string, update bool, verbose bool, envVars []string) e
 		}
 	}
 
+	if tr.testWork {
+		addSetup(func(env *testscript.Env) error {
+			fmt.Fprintf(os.Stderr, "temporary work directory for %s: %v\n", renderFilename(filename), env.WorkDir)
```

Comment:
nit: %s for WorkDir

Reply:
I will break myself of this habit one day. Thank you